ReactAndroid/src/main/java/com/facebook/react/views/webview/ReactWebViewManager.java

```diff
@@ -277,6 +278,11 @@ public void setUserAgent(WebView view, @Nullable String userAgent) {
     }
   }
 
+  @ReactProp(name = "mediaPlaybackRequiresUserAction")
+  public void setMediaPlaybackRequiresUserAction(WebView view, boolean requires) {
+    view.getSettings().setMediaPlaybackRequiresUserGesture(requires);
```

Comment:
The [Android docs](http://developer.android.com/reference/android/webkit/WebView.html#getSettings()) don't state that it can ever return `null` and there are 4 other established instances in the same class where `view.getSettings()` is used without checking for `null` so I'd say it's pretty safe not to check in this instance.